### ISSUE TITLE
Latest update seems to have broken Post Processing

### ISSUE THREAD

**Issue by ak2766** [SickChill/SickChill#4550]
This has been working well until the latest upgrade; I always upgrade from within the UI when prompted to.

Also, due to my lazy nature, where you see /pom, this is a symbolic link to /media/dlna/torrents/pom - :)

### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit:
MASTER/commit 9783c5f4de5d600d249705ce230cca53a9169f65

OS:
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 8.10 (jessie)
Release:        8.10
Codename:       jessie

Browser:
Google Chrome Version 65.0.3325.181 (Official Build) (64-bit)

What you did:
Copied files to my post processing folder and then proceeded to manually run post processing

What happened:
The files in the post processing folder were not moved to the appropriate TV folder

What you expected:
The files in the post processing folder should have been moved to the appropriate TV folder 

Logs:
```
2018-04-02 14:22:38 DEBUG    POSTPROCESSOR :: Starting new thread: POSTPROCESSOR
2018-04-02 14:22:38 INFO     POSTPROCESSOR :: Auto post processing task for /pom was added to the queue
2018-04-02 14:22:43 INFO     POSTPROCESSOR-AUTO :: Beginning auto post processing task: /pom
2018-04-02 14:22:43 DEBUG    POSTPROCESSOR-AUTO :: Processing in folder /media/dlna/torrents/pom
2018-04-02 14:22:43 INFO     POSTPROCESSOR-AUTO :: Processing /media/dlna/torrents/pom
2018-04-02 14:22:43 DEBUG    POSTPROCESSOR-AUTO :: Processing folder /media/dlna/torrents/pom
2018-04-02 14:22:43 DEBUG    POSTPROCESSOR-AUTO :: /media/dlna/torrents/pom : No processable items found in folder
2018-04-02 14:22:43 INFO     POSTPROCESSOR-AUTO :: Successfully processed
2018-04-02 14:22:43 INFO     POSTPROCESSOR-AUTO :: Auto post processing task for /pom completed
```

Below are some screenshots:
![selection_519](https://user-images.githubusercontent.com/6629646/38183194-b1ee0702-3681-11e8-9149-777f7b9ac208.png)
![selection_518](https://user-images.githubusercontent.com/6629646/38183195-b804291e-3681-11e8-9bd4-555e3cf71e80.png)

**Comment by ak2766** [comment]
OK, I think I have found the main issue.

I had previously downloaded .avi versions of the show files and when the x264 versions appeared, I proceeded to download them.  However, SickRage appears to not be able to overwrite files which have a status of 'Downloaded `Unknown`':

![selection_520](https://user-images.githubusercontent.com/6629646/38184597-21e7bb0c-368c-11e8-8867-b43ec23f417c.png)

Even after changing the status to either Skipped or Wanted, post processing still barfs on them.
